presto-main/src/main/java/io/prestosql/sql/analyzer/FeaturesConfig.java

```diff
@@ -814,6 +823,18 @@ public FeaturesConfig setExchangeCompressionEnabled(boolean exchangeCompressionE
         return this;
     }
 
+    public DataIntegrityVerification getExchangeDataIntegrityVerification()
+    {
+        return exchangeDataIntegrityVerification;
+    }
+
+    @Config("exchange.data-integrity-verification")
```

Comment:
```suggestion
    @Config("exchange.data-integrity-protection")
```

?

Reply:
I like verification more. Or `data-integrity-checking`. 
If you decide on different name, change the variables/methods/.. too of course.